src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
```

Comment:
For the future, remove the first empty line in all classes/interfaces/methods where it is present